File: src/main/java/pl/oczadly/kotlin_app/hackyeah2025/Hackyeah2025Application.java
package pl.oczadly.kotlin_app.hackyeah2025;

import io.swagger.v3.oas.annotations.OpenAPIDefinition;
import io.swagger.v3.oas.annotations.servers.Server;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@OpenAPIDefinition(servers = @Server(url = "/", description = "Default Server URL"))
@SpringBootApplication
public class Hackyeah2025Application {

    public static void main(String[] args) {
        SpringApplication.run(Hackyeah2025Application.class, args);
    }
}

File: src/main/kotlin/pl/oczadly/kotlin_app/hackyeah2025/Hackyeah2025Application.kt
package pl.oczadly.kotlin_app.hackyeah2025

import io.swagger.v3.oas.annotations.OpenAPIDefinition;
import io.swagger.v3.oas.annotations.servers.Server;
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@OpenAPIDefinition(servers = [Server(url = "/", description = "Default Server URL")])
@SpringBootApplication
class Hackyeah2025Application

fun main(args: Array<String>) {
	runApplication<Hackyeah2025Application>(*args)
}
